.travis.yml

```diff
@@ -5,16 +5,48 @@ env:
   - BUILD_DEPENDS="geomet>=0.1,<0.3 six>=1.9 futures<=2.2.0 Cython>=0.20,!=0.25,<0.30"
   - TEST_DEPENDS=""
   - UNICODE_WIDTH=32
+  - MB_ML_VER=1
```

Comment:
Required after the multibuild upgrade.  Default got bumped up to manylinux2014 and current versions of that image don't include Python 2.7 and 3.5 interpreters.